admin-src/DiagnoseInfo/components/Responses/index.js

```diff
@@ -0,0 +1,50 @@
+import React, { useEffect, useState } from 'react'
+import { isEmpty } from 'lodash'
+import { queryAllResponsesForRequest } from '../../../services/database/queryAllResponsesForRequest'
+import { firebaseTimestampToMoment } from '../../../utils/date'
+import Response from '../Response'
+import './styles.css'
+
+const Responses = ({ diagnose }) => {
+  const [responses, setResponses] = useState([])
+  const scrollToBottomRef = React.createRef()
+
+  useEffect(() => {
+    const buildResponses = async () => {
+      const responses = await queryAllResponsesForRequest(diagnose.userUID, diagnose.id)
+
+      const responsesComponents = responses.map((response, key) =>
+        <Response
+          key={key}
+          text={response.answer}
+          date={firebaseTimestampToMoment(response.createdAt).format('LLL')}
+          isAdmin={response.answeredBy !== diagnose.username}
+        />
+      )
+
+      setResponses(responsesComponents.reverse())
+    }
+
+    buildResponses()
+  }, [])
+
+  useEffect(() => {
+    const scrollToBottom = () => {
+      scrollToBottomRef.current.scrollTop = scrollToBottomRef.current.scrollHeight
+    }
+
+    if (scrollToBottomRef.current) {
+      scrollToBottom()
+    }
+  }, [scrollToBottomRef])
```

Comment:
This will execute only once. A ref is an object *that never changes*, only its `current` property changes. You can pass a function to a `ref` prop. The function will get called with a reference to the element. Use that to scroll to the bottom on initialization. Otherwise, consider executing this effect every time `responses` changes.

```javascript
<div ref={ref => { /* do something with ref */ }} />
```